src/main/resources/org/datadog/jenkins/plugins/datadog/DatadogBuildListener/global.jelly

```diff
@@ -23,9 +23,13 @@
     <f:validateButton title="${%Test Hostname}" progress="${%Testing...}"
                       method="testHostname" with="hostname" />
     <f:entry title="Blacklisted Jobs"
-             description="A comma-separated list of job names that should not monitored. (eg: susans-job,johns-job,prod-release)." >
+             description="A comma-separated list of job names that should not monitored. (e.g.: susans-job,johns-job,prod-release)." >
```

Comment:
Let's add to this description something like `This takes precedent over whitelist.`